tests/haskell_module/nested/Branch/Left/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("@rules_haskell//haskell/experimental:defs.bzl", "haskell_module")
+
+haskell_module(
+    name = "Module",
+    src = "Module.hs",
+    src_strip_prefix = "/tests/haskell_module/nested",
```

Comment:
I'm not sure I understand the question. GHC needs an import path here, otherwise, it won't find the interface file when another module imports `Branch.Left.Module`. I.e. if you remove this `src_strip_prefix`, then 
```
$ bazel build //tests/haskell_module/nested:LeafModule
```
fails with
```
tests/haskell_module/nested/LeafModule.hs:3:1: error:
    Could not find module ‘Branch.Left.Module’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
  |
3 | import Branch.Left.Module
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
Target //tests/haskell_module/nested:LeafModule failed to build
```

Reply:
Oh, I understand. And since we're in a subdirectory of that directory here, it has to be workspace-relative. My mistake.